plugins/datetime/msd-datetime-mechanism.c

```diff
@@ -450,199 +497,360 @@ msd_datetime_mechanism_set_timezone (MsdDatetimeMechanism  *mechanism,
                         code = MSD_DATETIME_MECHANISM_ERROR_GENERAL;
 
                 error2 = g_error_new (MSD_DATETIME_MECHANISM_ERROR,
-                                      code, "%s", error->message);
+                                code, "%s", error->message);
```

Comment:
If you do not make any changes to the code, please do not format it, as you will increase the number of lines to review.